modules/acquia_cms_headless/tests/src/Functional/HeadlessModeEnablement.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Drupal\Tests\acquia_cms_headless\Functional;
+
+/**
+ * Tests Headless mode enablement.
+ *
+ * @group acquia_cms
+ * @group acquia_cms_headless
+ * @group medium_risk
+ * @group push
+ */
+class HeadlessModeEnablement extends HeadlessTestBase {
+
+  /**
+   * Test Enable Headless mode and Enable Next.js starter kit.
+   */
+  public function testHeadlessMode(): void {
+    $account = $this->createUser();
+    $account->addRole('administrator');
+    $account->save();
+    $this->drupalLogin($account);
+    $assertSession = $this->assertSession();
+
+    // Click on Setup manually button to close the modal,
+    // note there are two button with same label 'Setup manually'.
+    $this->drupalGet('/admin/tour/dashboard');
+    $acmsWelcomeModal = $assertSession->waitForElementVisible('css', '.acms-welcome-modal');
+    $btnPanes = $assertSession->elementExists('css', '.ui-dialog-buttonpane', $acmsWelcomeModal);
+    $assertSession->buttonExists('Setup Manually', $btnPanes)->press();
+
+    // Test if Headless block is available.
+    $headlessForm = $assertSession->elementExists('css', '#acquia-cms-headless-form');
+    $headlessSummary = $assertSession->elementExists('css', '.claro-details__summary');
+    $this->assertEquals('Headless', $headlessSummary->getText());
+    $assertSession->elementExists('css', '.claro-details__summary', $headlessForm)->press();
+
+    // Test if next js gets enabled or not.
```

Comment:
@deepakmishra2 can we include Next.js starter kit enablement in HeadlessHybridModeTest? you can rename HeadlessHybridModeTest to HeadlessModesTest and include it.
cc @vishalkhode1 @apathak18

Reply:
done with these changes